index.html

```diff
@@ -29,57 +31,56 @@ <h1 class="w3-display-topmiddle w3-container">Trick</h1>
     <!-- Host will switch to page where HOST can set up game with OPTIONS -->
     <!-- Guest player will be switching to a new page to enter the CODE from HOST  -->
     <main>
-        <section id='landing-page' class='page hidden'>
+        <section id='landing-page' class='page w3-display-container w3-center'>
             <h2>Welcome to Trick Trivia!</h2>
             <span id="player-name">
-                <label for="player-name-input">Player Name: </label>
-                <input id="player-name-input" type="text">
+                <label for="player-name-input">Player: </label>
+                <input id="player-name-input" type="text" placeholder=" Sign In">
```

Comment:
No space in front of sign in. Also, if you are going to do place holder text, it sould be more consistant. You cant just have it here and not everyplace else. Unless there is a really good reason to do it here. And if you do, then you really don't need the labe and the place holder text. I think saying sign in might be a bit confusing. It might be better to just not have the placeholder.

Reply:
Got it. I haven't seen another input field yet but I haven't been to all pages of the game. In theory I would conform those fields but I see your point about the redundancy.